x-pack/plugins/translations/translations/ja-JP.json

```diff
@@ -6841,7 +6841,6 @@
     "xpack.lens.indexPatterns.filterByTypeLabel": "タイプでフィルタリング",
     "xpack.lens.indexPatterns.noFieldsLabel": "このインデックスパターンにはフィールドがありません。",
     "xpack.lens.indexPatterns.noFilteredFieldsLabel": "現在のフィルターと一致するフィールドはありません。フィルターまたは時間範囲を変更してください。",
```

Comment:
I think you've changed the translations for `emptyFieldsWithDataLabel`, `noFieldsLabel` and `noFilteredFieldsLabel` without deleting them from the translation file. We usually do that to indicate they need to be re-translated.

Reply:
So running the translations script doesn't check for changed defaults?

Reply:
Exactly

Reply:
Good to know!! I removed any changed translation tokens